superset/models/helpers.py

```diff
@@ -1162,6 +1162,12 @@ def filter_values_handler(  # pylint: disable=too-many-arguments  # noqa: C901
         def handle_single_value(value: Optional[FilterValue]) -> Optional[FilterValue]:
             if operator == utils.FilterOperator.TEMPORAL_RANGE:
                 return value
+
+            if (
```

Comment:
there's long-standing issue where someone copied a bunch of methods/logic between the two files in this PR, and we really need to fix this ... @betodealmeida and myself have spoken about this and we're meaning to fix this once and for all. Not sure what exactly it means for this PR, but I'd say if we're going to fix a thing here let's dedup at least the method we're touching here (`handle_single_value`)